src/Builders/Paths/RouteParams/RouteParamsBuilder.php

```diff
@@ -23,6 +25,16 @@ public static function build(Route $route): array
                 ->required();
         }
 
+        if (isset($route->defaults['resource_type']) && str_contains($route->getName(), 'index')) {
```

Comment:
```suggestion
        $resourceType = $route->parameter(\LaravelJsonApi\Laravel\Routing\Route::RESOURCE_TYPE);
        if ($resourceType !== null && str_contains($route->getName(), 'index')) {
```

Als de defaults wordt overschreven gaat dit niet goed. Aangezien de 'resourceType' methode op de Route class een exception throwt als deze niet gezet is lijt me dit een beter alternatief?

Reply:
@amooij sorry, deze opmerking klopt niet en heb ik verkeerd gezien. Als ik dit nu test gaat dit stuk. Kan je dit terugzetten naar hoe dit was? Je code kloptte gewoon.